src/node/rpc/member_frontend.h

```diff
@@ -2354,6 +2352,11 @@ namespace ccf
           "{}\n export default (input) => validate(input);",
           constitution.value());
 
+        js::Runtime rt;
+        js::Context context(rt);
+        rt.add_ccf_classdefs();
+        js::populate_global_ccf(nullptr, std::nullopt, nullptr, context);
```

Comment:
More of a comment:

I thought this line meant we were now exposing a KV access to `validate`, where we hadn't previously. After reading more closely this isn't the case - the first `nullptr` arg results in us _not_ creating a `kv` field on the `ccf` object. I think these relationships aren't very clear, and we should more verbose initialisers so its obvious what's accessible in each JS context (something like `js::add_ccf_node()`, `js::add_ccf_kv()`).